### ISSUE TITLE
HTML needs a lang attribute 

### ISSUE THREAD

**Issue by jennah2121** [fac-13/Team-GIPHT-Todo#35]
Well done on the accessibility rating of 94%! The accessibility audit says that your HTML element is missing its lang attribute.
